engine/apps/slack/scenarios/ancillary.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
not sure if ancillary is a good name for the file, it should reflect the purpose of the scenario